dotnet-3.1/.gitignore

```diff
@@ -0,0 +1,454 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Can we minimise this file to the bare minimum? Looks like there are too many files in here.

Reply:
This is the default .gitignore file created by the .NET CLI. I can, however, try and remove some of the irrelevant files.